legislative/Q2994125/popolo-m17n.json

```diff
@@ -0,0 +1,567 @@
+{
+  "persons": [
+
+  ],
+  "organizations": [
+
+  ],
+  "areas": [
+    {
+      "id": "Q16",
+      "identifiers": [
+        {
+          "scheme": "MS_FB",
+          "identifier": "country:ca"
+        },
+        {
+          "scheme": "wikidata",
+          "identifier": "Q16"
+        }
+      ],
+      "associated_wikidata_positions": [
+        "Q14212"
+      ],
+      "type": {
+        "lang:en_CA": "country",
+        "lang:fr_CA": "pays"
+      },
+      "name": {
+        "lang:en_CA": "Canada",
+        "lang:fr_CA": "Canada"
+      },
+      "parent_id": null
```

Comment:
These `parent_id` values being null is a problem; it's due to something I really should have mentioned before, but forgot (sorry!) for adding city boudaries, we also need to add an Ottawa boundary made up by dissolving together all these ward boundaries to the `cities` Shapefile. It's probably worth us going over this together, screen-sharing, and updating the instructions at the same time.

Another thing I'll create a ticket for is to make sure the build errors with a helpful error if anything other than the country-level area has parent_id null.